src/data_collector.py

```diff
@@ -0,0 +1,52 @@
+import aiohttp
+import asyncio
+import boto3
+import json
+
+STATUS_NODES = [
+    'node.explorer.hathor.network',
+    'node1.mainnet.hathor.network',
+    'node2.mainnet.hathor.network'
+]
+
+def process_node_status(node_status: dict):
+    session = boto3.session.Session()
+    lamb = session.client("lambda", endpoint_url="http://localhost:3002")
+    lamb.invoke(
+        InvocationType='Event',
+        FunctionName='hathor-explorer-service-dev-network_data_aggregator',
+        Payload=json.dumps(node_status)
+    )
+
+async def request_node(session: aiohttp.ClientSession, url: str):
+    try:
+        async with session.get(url) as resp:
+            status = await resp.json()
+            process_node_status(status)
+    except:
+        pass
```

Comment:
Maybe we shouldn't hide exceptions to not compromise observability of what is happening in the service.

Wouldn't we want to see in the logs if the request to some node is not succeeding, for example?

I think logging the exception here is a good idea.

Reply:
Agree! Can we discuss about logging?